### ISSUE TITLE
Action not throwing exitcode anymore

### ISSUE THREAD

**Issue by BennyTheSen** [ibiqlik/action-yamllint#28]
I used the action previously and it was throwing an exitcode (1) when finding errors with the default parameters for the action.

When I'm running the action at the moment it's not generating an exitcode anymore:

```
Run ibiqlik/action-yamllint@v3.0.3
  with:
    strict: true
    format: github
    no_warnings: false
  env:
    tf_version: 1.0.2

======================
= Linting YAML files =
======================
Warning: [truthy] truthy value should be one of [false, true]
Error: [line-length] line too long (86 > 80 characters)
Error: [hyphens] too many spaces after hyphen
Error: syntax error: expected <block end>, but found '<block mapping start>' (syntax)
Warning: [comments-indentation] comment not indented like content
```

Here is the workflow config:

```
      - name: "YAML lint"
        uses: ibiqlik/action-yamllint@v3.0.3
        with:
          strict: true
```
I tried it with the github and default format, same result.

**Comment by ibiqlik** [comment]
😱 oh my, I'm on it.

**Comment by ibiqlik** [comment]
@BennyTheSen Thanks a lot for reporting it 🙇  It should be fixed now in v3, v3.0 and v3.0.4

**Comment by BennyTheSen** [comment]
Yes testet it again with v3, working like intended. Thank you for the fast fix.
